pkg/snapshot/overlay.go

```diff
@@ -299,8 +299,9 @@ func (o *snapshotter) Prepare(ctx context.Context, key, parent string, opts ...s
 		return nil, err
 	}
 
-	// NOTE: If the image is in overlaybd format, the baselayer will be
-	// the metadata(in small size) and should not be fetched on-demand.
+	// If the layer is in overlaybd format, only its metadata needs to be downloaded and saved into snapshot dir.
```

Comment:
metadata comes from annotations of manifests, and already been downlaoded with manifest together, not downloaded here.

Reply:
-> backstore information